src/test/frontend/deploy/deploy_controller_test.js

```diff
@@ -0,0 +1,52 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import DeployController from 'deploy/deploy_controller';
+import deployModule from 'deploy/deploy_module';
+
+describe('Deploy controller', () => {
+  let vm;
+
+  beforeEach(() => {
+    angular.mock.module(deployModule.name);
+
+    angular.mock.inject(($resource, $log, $state, $mdDialog) => {
+      let namespacesMock = {namespaces: []};
+      vm = new DeployController($resource, $log, $state, $mdDialog, namespacesMock);
```

Comment:
Use ctrl instead of vm. All other parts of the code use ctrl :) Consistency and code is what matters in long term projects. You'll see benefits of this soon, I hope :)